### PR TITLE
Add missing properties to case and workflow factory, add test

### PR DESCRIPTION
**Pull request by czosel**
... and also update the schema.

### CODE REVIEW

**Comment by czosel** [comment]
@kaldras ready for review

File: package.json [hunk 1]
```diff
@@ -92,6 +92,7 @@
     "eslint-plugin-ember": "6.1.0",
     "eslint-plugin-node": "8.0.1",
     "eslint-plugin-prettier": "2.6.0",
+    "graphql-cli": "3.0.11",
```
Comment (kaldras) [inline package.json]: Where do you use this? Or why is this needed?

Comment (czosel) [reply package.json]: This is needed to run `graphql schema-update` - i forgot to migrate this dependency from ember-caluma-utils.

Comment (kaldras) [reply package.json]: Ok, thanks for the explanation :+1:
